services/event-application-service/src/pubsub_controller/pubsub_controller.cc

```diff
@@ -0,0 +1,77 @@
+#include "pubsub_controller.h"
+
+#include "glog/logging.h"
+
+namespace pubsub = ::google::cloud::pubsub;
+namespace cloud = ::google::cloud;
+
+struct FormQuery {
+  static constexpr std::string_view topic = "form_get";
+  std::string data;
+
+  std::string ToString() const { return data; }
+};
+
+namespace eas::pubsub_controller {
+
+PubSubController::PubSubController(
+    std::unique_ptr<IConnFactory> conn_factory,
+    std::shared_ptr<queries::IQueryHandler> query_handler,
+    std::shared_ptr<commands::ICommandHandler> cmd_handler)
+    : connection_factory_{std::move(conn_factory)},
+      query_handler_{query_handler},
+      cmd_handler_{cmd_handler},
+      pubsub_thread_pool_{ThreadPool(6)} {}
+
+// 6 threads should be enough as simple initial implemetation.
+PubSubController::PubSubController(
```

Comment:
Probably, but still let's move it to const

Reply:
Okey